### PR TITLE
Update flutter code samples as per library refactor 

### PR DESCRIPTION
**Pull request by Amplifiyer**
*Issue #, if available:* 
aws-amplify/amplify-flutter#274
aws-amplify/amplify-flutter/pull/273
aws-amplify/amplify-flutter/pull/275

*Description of changes:*

@Amplifiyer
Update flutter code samples as per library refactor 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jamesonwilliams — APPROVED** [review]

File: docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md [hunk 1]
```diff
@@ -34,10 +31,9 @@ class MyAmplifyApp extends StatefulWidget {
         AmplifyAuthCognito authPlugin = AmplifyAuthCognito();
         AmplifyAnalyticsPinpoint analyticsPlugin = AmplifyAnalyticsPinpoint();
 
-        amplifyInstance.addPlugin(
-            authPlugins: [authPlugin], analyticsPlugins: [analyticsPlugin]);
+        Amplify.addPlugins([authPlugin, analyticsPlugin]);
```
Comment (jamesonwilliams) [inline docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md:34]: Should we just inline the plugin constructors? That is, remove lines 31-32 and simplify to:
```dart
Amplify.addPlugin(AmplifyAuthCognito());
Amplify.addPlugin(AmplifyAnalyticsPinpoint());
```

Alternately, if you want to show-off the list-style method, we could do:
```dart
Amplify.addPlugins([AmplifyAuthCognito(), AmplifyAnalyticsPinpoint()]);
```
?

Same suggestion throughout the PR

**Review by joekendal — APPROVED** [review]

File: docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md (hunk 1 above)
Comment (Amplifiyer) [reply docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md]: Good suggestion. Updated the PR.

**Review by joekendal — APPROVED** [review]

File: docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md [hunk 2]
```diff
@@ -1,21 +1,15 @@
...
+Amplify.addPlugins([AmplifyAuthCognito(), AmplifyAnalyticsPinpoint()]);
 ```
 
 Make sure that the amplifyconfiguration.dart file generated in the project setup is included and sent to Amplify.configure: 
 
 ```dart 
 import 'amplifyconfiguration.dart';
 
-Amplify.configure( amplifyconfig )
+Amplify.configure(amplifyConfig)
```
Comment (Ashish-Nanda) [inline docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md:12]: This is the only code example where we are not showing the import statements for the packages. Should we add that?

**Review by Ashish-Nanda — APPROVED** [review]
Left one minor comment, otherwise looks good.

File: docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md (hunk 2 above)
Comment (Amplifiyer) [reply docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md:12]: Good point, while checking it seems the getting started guide is all over the place. DataStore and API follow the analytics format and show the sample code, while others completely skip it (Storage, Auth)

I'll match it with DataStore/API for now, we may have to do some restructure later on.

**Review by joekendal — APPROVED** [review]

**Pull request merged**